packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -156,6 +157,14 @@ const getRouterRenderer = ({
 		return redirect;
 	}
 
+	// Check launch darkly flags to determine if we should
+	// be rendering the new swarm global styles as well.
+	// The new global styles must be loaded after the old css styles
+	const ldStyleFlag = initialState.flags['new-swarm-assets'];
```

Comment:
I'd really like to avoid putting application-specific code into this function - e.g. any reference to a particular feature flag name -  because it doesn't solve for the general case that the platform should aim to support.

Instead, I was wondering if we could update how `cssLinks` is read, e.g. by allowing it to be _either_ an array _or_ a function. If it's a function (`typeof cssLinks === 'function'`), then it could be treated as a selector that takes `initialState` as an argument and returns an array of links. That would allow the caller to essentially compose `cssLinks` in whatever way it wants based on `state`, just like a `mapStateToProps` function.

There might be some other options, but essentially we want to be able to make it the responsibility of the caller to provide the right assets in one shot rather than composing internally if possible.